modules/apps/headless/headless-admin-taxonomy/headless-admin-taxonomy-test/src/testIntegration/java/com/liferay/headless/admin/taxonomy/resource/v1_0/test/KeywordResourceTest.java

```diff
@@ -30,12 +41,102 @@
 @RunWith(Arquillian.class)
 public class KeywordResourceTest extends BaseKeywordResourceTestCase {
 
-	@Before
 	@Override
-	public void setUp() throws Exception {
-		super.setUp();
+	@Test
+	public void testGetKeywordsRankedPage() throws Exception {
```

Comment:
why do you need to overwrite this one?

Reply:
To specify the groupId in the `keywordResource.getKeywordsRankedPage`. If not the test fails getting more results than they expect, as they are keywords for both, the test site and the test asset library